res/database/Default/database_news.xml

```diff
@@ -1439,10 +1679,12 @@ Her normal body size is expected to be reached in a week.</en>
 			<title>
 				<de>Welt lacht über ${animal_${index}}-%minister%</de>
 				<en>World laughs at ${animal_${index}} %minister%</en>
+				<pl>Świat śmieje się z urzędnika i jego rozporządzenia.</pl>
 			</title>
 			<description>
 				<de>Die ganze Welt lacht über unseren %minister%, der die Haltung ein${animal_gen_${index}} zur Pflicht machen will.</de>
 				<en>The whole world is laughing at our %minister% who wants to make keeping a${animal_gen_${index}} mandatory.</en>
+				<pl>Cały świat śmieje się z naszego urzędnika, który chce wprowadzić obowiązek posiadania ${animal_gen_${index}}.</pl>
```

Comment:
minister variable not used on purpose?

Reply:
I've changed this. %minister% in front of sentence, because Capital letter.